src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -374,7 +395,7 @@ function getStartTextToReplace(type: MarkdownButtonType): string[] {
 function getStartTextToInsert(type: MarkdownButtonType): string {
 	switch (type) {
 		case MarkdownButtonType.BOLD:
-			return '**';
+			return '<b>';
```

Comment:
This is supposed to be the markdown tags, why are you changing it?

Reply:
I did this to test the addition of markdown instead. Yes, for markdown purposes that needs to be `**`.